sw/tools/airframe_editor/xml_airframe.py

```diff
@@ -76,13 +92,18 @@ def reorganize_airframe_xml(airframe_xml):
     find_and_add_sections_with_name(airframe_xml, airframe, "XSENS")
 
     find_or_add_group(airframe_xml, airframe, "GAINS")
+
     # Fixedwing
-    find_and_add_sections_with_name(airframe_xml, airframe, "HORIZONTAL CONTROL")
+    find_and_add_sections_with_name(airframe_xml, airframe,
```

Comment:
I have already had at look at that (to understand how the Paparazzi Center works) since I currently better understand the logic in Python than in OCaml ;-) I will have more time in September to spend on it and I am definitely interested in trying to improve it. Can you be a bit more specific on which panels to show/hide and when? Cheers.